sdk/src/ssi/credential.go

```diff
@@ -0,0 +1,51 @@
+package ssi
+
+import (
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/goccy/go-json"
+)
+
+// SignVerifiableCredentialJWT takes in a key ID, key type, private key, and a verifiable credential
+// The keyID and KeyType are used to reconstruct a go-friendly private key to be used for signing
+// the credential, which will be packaged as a JWT according to the VC-JWT 1.0 specification.
+// The resulting is returned as a string representation of a JWT.
+func SignVerifiableCredentialJWT(keyID, keyType string, privateKey, vcJSONBytes []byte) string {
```

Comment:
This is excellent, and I was working on a really similar approach locally! (but my go skills are not great 😄)

One question that I have that I'd love to get other's opinions on:
Because we're just working with raw VC bytes on the sign/verify functions, is there a benefit to having the `VerifiableCredentialMobile` struct?

Reply:
Same question from me

Reply:
I would actually rather it go away and unwrap it internally somewhere mobile people don't care about it and just send us a panic with an exception if unwrapping it goes wrong

Reply:
No need for it. However you will need some representation of the VC after parsing it out from the JWT. I believe the Veramo and other libs do this. A few reasons;
- Render credentials properly
- Able to introspect certain fields, needed for different interactions (e.g. you may want to hide all credentials with an expiration date in the past)
- Able to respond to presentation requests with selective disclosure - meaning selecting fields in a VC you wish to share

Reply:
We can handle all of that on the JS side but whever we use the Go SDK it should be either primitives or binary, and then you unwrap however you want to inside of the Go.

For instance, I think `SsiStringArray` probably needs to go away

Reply:
@shamilovtim Agreed! I think we can clean up the interface quite a bit after this lands.